edit/service.vue

```diff
@@ -92,12 +93,26 @@ export default {
       sessionAffinityActionOptions: Object.values(
         SESSION_AFFINITY_ACTION_VALUES
       ),
+      formRulesets:         [],
+      boundValidationPaths: ['spec']
     };
   },
 
   computed: {
     ...mapGetters(['currentCluster']),
 
+    tabErrors() {
```

Comment:
You're right, it makes way more sense for the triangle to stay visible.

Since the tab states are currently limited to "active" or "inactive", I went ahead and just removed the not active condition on these since adding additional states such as "blurred" seems a bit overkill here.